sched/sagaLog.thrift

```diff
@@ -0,0 +1,16 @@
+
+struct Command {
+  1: list<string> argv,
+  2: map<string, string> envVars,
+  3: i64 timeout,
+  4: string snapshotId,
+}
+
+struct TaskDefinition {
+  1: Command command,
+}
+
```

Comment:
Can you add struct Job which mimics Job struct in definitions.go?  We'll need the Id to be stored in the WorkQueue & the SagaLog when we store this information.